modules/rtgtools/vcfeval/main.nf

```diff
@@ -43,6 +47,8 @@ process RTGTOOLS_VCFEVAL {
         --template=$sdf \\
         --threads=$task.cpus \\
 
+    mv ${prefix}/* .
```

Comment:
What's the purpose of this?

Reply:
It moves all files from the folder created by the tool, to the main directory of the process. This way it's easier to define them for the output channels

Reply:
Better to modify the output glob instead

Reply:
use wildcards to recurse into the directory, as in the suggestions

Reply:
That's useful indeed, I forgot about that! Thanks a lot for the help.